system/vmware-monitoring/templates/servicemonitors/infra-frontend.yaml

```diff
@@ -0,0 +1,43 @@
+{{- $root := . }}
+{{- range $target := .Values.global.targets }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    ccloud/support-group: observability
+    prometheus: infra-frontend 
+  name: {{ include "prometheusVMware.name" (list $target $root) }}
+  namespace: vmware-monitoring
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 60s
+    path: federate
+    port: http
+    scheme: http
+    scrapeTimeout: 55s
+    params:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 60s
+    path: federate
+    port: http
+    scheme: http
+    scrapeTimeout: 55s
+    params:
+      match[]:
+        - '{job="vrops-exporter",__name__!~"^vrops_virtualmachine_.*", __name__!~"^(up|ALERTS.*|scrape.+)"}'
+        - '{job="vrops-exporter",__name__=~"^vrops_virtualmachine_.*", vccluster=~".*management.*", __name__!~"^(up|ALERTS.*|scrape.+)"}'
+        - '{job="vrops-inventory-exporter", __name__!~"^(up|ALERTS.*|scrape.+)"}'
+    metricRelabelings:
+    - action: replace
+      sourceLabels: [__address__]
+      targetLabel: region
+      regex: prometheus-vmware.(.+).cloud.sap
+      replacement: $1
```

Comment:
Do we really need this?

Reply:
yeah absolutely right, wouldn't match anyway